src/encoded/tests/test_types_biosample.py

```diff
@@ -116,10 +116,13 @@ def test_biosample_summary(testapp,
     testapp.patch_json(biosample_1['@id'], {'donor': donor_1['@id'],
                                             "biosample_ontology": liver['uuid'],
                                             "preservation_method": "cryopreservation",
+                                            "post_nucleic_acid_delivery_time": 3,
+                                            "post_nucleic_acid_delivery_time_units": "week",
                                             'treatments': [treatment_5['@id']]})
     res = testapp.get(biosample_1['@id']+'@@index-data')
     assert res.json['object']['summary'] == (
-        'Homo sapiens male child (10 days) liver tissue treated with ethanol, preserved by cryopreservation')
+        'Homo sapiens male child (10 days) liver tissue treated with ethanol,'
```

Comment:
This is a non-code review comment. I thought living tissue was preserved in formaldehyde?

Reply:
The properties are arbitrary. The summary printout just depends on the properties patched just a few lines prior (L115-121).

Reply:
@zoldello there are many ways to preserve tissue - and in this case it's no longer "living".  Formaldehyde specifically interferes with many downstream biology experiments.